README.md

```diff
@@ -1,2 +1,156 @@
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Mobile first approach**
+- **HTML & CSS Linters**
+- **Media Queries**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Demo link is coming soon](#)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+### Prerequisites
+
+In order to run this project you need:
+- Web Browser (Chrome, Edge, Safari, etc.)
+- Text Editor (Visual Studio Code, etc.)
+
+If you like to try Linters results, you should need to follow this guide: https://github.com/microverseinc/linters-config/tree/master/html-css
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd local-folder
+  git clone git https://github.com/alexansaa/Personal_Portafolio.git
+```
+
+### Install
+
+This project won't need further installation
+
+### Usage
+
+To run the project double click the "index.html" file.
+
+### Run tests
+
+To test Linters, after you have completed the guided tutorial above (the GitHub link), push the code to your new repository and check the Linters test in the project repository webpage or use GitHub for your desktop.
+
+There are no extra tests for the code itself. If you run the code, you should see the webpage.
+
+### Deployment
+
+Deploy this project on any websever.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Alexander Saavedra**
+
+- GitHub: [@alexansaa](https://github.com/alexansaa)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/alexander-saavedra-2803b1b6/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Web Page Optimized for wither screen sizes**
+- [ ] **Larger Images Implementation**
+- [ ] **New CSS Styles**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give me a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank every who continues supporting me.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Kindly fix this link. It is currently broken and makes your `README` looks unprofessional
```suggestion
This project is [MIT](./LICENSE) licensed.
```
This is the current situation where I tried to access it.
![Screenshot 2023-06-02 at 05 35 03](https://github.com/alexansaa/Personal_Portafolio/assets/112261237/4de4805b-e414-491f-8e72-09dae58aad57)